docs/en/integrations/data-visualization/looker-studio-and-clickhouse.md

```diff
@@ -121,3 +121,13 @@ Finally, you should be able to see the introspected table structure and adjust t
 <br/>
 
 Now you can proceed with exploring your data or creating a new report! 
+
+## ClickcHouse Cloud Connection
+
+When using ClickHouse Cloud, you must first file a support case to enable the MySQL feature. Our support team will enable this feature, which then creates a dedicated user for the MySQL protocol.
```

Comment:
Shall we avoid "must" in this case? And protocol -> interface (https://clickhouse.com/docs/en/interfaces/mysql)

```suggestion
When using ClickHouse Cloud, create a support case to enable the MySQL feature. Our support team will enable this feature, which then creates a dedicated user for the MySQL interface.
```